README.md

```diff
@@ -237,7 +255,10 @@ For example, the following command line will copy audio data starting at offset
 ```
 python3 wave-recovery-tool-master/waverecovery.py -r -s 96000 -b 24 -o 153608 -e -334
 ```
+### GUI
+
+Just run: ``` Python3 waverecovery-gui.py ```
```

Comment:
To create a code block, the three dashes have to be on separate lines. Please align with the rest of the documentation: `python3` instead of `Python3`. Maybe we should add two different paragraphs and commands for Windows and Mac/Linux. Finally, align the file name according to the new file name of your `.py` file (see comment in `waverecovery-gui_modern.py`)